frontend/src/App.tsx

```diff
@@ -1,31 +1,26 @@
 import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '../../../../../vite.svg';
 import './App.css';
+import MockPage from './components/mockPage/MockPage';
+import LoginPage from './components/loginPage/LoginPage';
+
+
+interface IUser {
```

Comment:
It good to decide on this stage are we going to use types or interfaces, I think now with the current types' capabilities we can use types.

Reply:
i think we can go with Types everywhere

Reply:
I use both generally but Types will be fine.

Reply:
Let's stick to types then.